### DISCUSSION TITLE
Is there a way to keep certain words or phrases in their original form?

### DISCUSSION THREAD

**Discussion by invalidname1911** [openai/whisper#2271] [Q&A]
im trying to use whisper to translate japanese recorded video into transcript, and the video is used for education purpose.

However, If i want to specifiy some word doesn't need translation, for example, the expected output is something like this:
"こんにちは" is "good morning". We want some part of our paragraph as the orginal it is.

However the whisper would translate the whole sentence: (chinese language in transcription video)
the inital input:
早晨日文是「こんにちは」
input after using whisper:
Morning in Japanese is “morning”

Is there a way to keep certain words or phrases in their original form?

**Comment by glangford** [comment]
> Is there a way to keep certain words or phrases in their original form?

whisper doesn't generate the "original form" unless you request to transcribe the audio (rather than translate). You may want to post-process a translation and a transcription, see for example

* https://github.com/openai/whisper/discussions/1876
